### PR TITLE
OHARA-1145 : Update StreamApp Action API of Swagger Doc and Add Inter…

### PR DESCRIPTION
**Pull request by saivirtue**
- 更新configurator.yaml (swagger)
- 增加StreamApp action (start / stop)的interface
- 修改StreamApp List Page : upload 成功時將temp file刪除

### CODE REVIEW

File: ohara-configurator/src/main/scala/com/island/ohara/configurator/route/StreamRoute.scala [hunk 1]
```diff
@@ -205,49 +206,27 @@ private[configurator] object StreamRoute {
...
-
-                // TODO: use LOG instead...by Sam
-                System.out.println(s"command : $dockerCmd")
-                complete(if (Process(dockerCmd).run.exitValue() == 0) StatusCodes.OK else StatusCodes.BadRequest)
+                complete(StreamActionResponse(id, StreamAppState.RUNNING))
               }
             }
           } ~
-          // stop streamapp
+          //TODO : implement stop action...by Sam
```
Comment (chia7712) [inline ohara-configurator/src/main/scala/com/island/ohara/configurator/route/StreamRoute.scala]: Do you plan to complete this job on 0.2?

File: ohara-client/src/main/scala/com/island/ohara/client/configurator/v0/StreamApi.scala [hunk 2]
```diff
@@ -59,13 +59,40 @@ object StreamApi {
     override def kind: String = "streamApp"
   }
 
+  sealed abstract class StreamAppState(state: String)
+  object StreamAppState {
+    case object UNASSIGNED extends StreamAppState("UNASSIGNED")
+    case object RUNNING extends StreamAppState("RUNNING")
+    case object FAILED extends StreamAppState("FAILED")
+    case object STOP extends StreamAppState("STOP")
+    val values = Seq(RUNNING, FAILED, STOP)
```
Comment (chia7712) [inline ohara-client/src/main/scala/com/island/ohara/client/configurator/v0/StreamApi.scala]: miss UNASSIGNED?

**Comment by jackyoh** [comment]
retry kafka

**Comment by saivirtue** [comment]
retry

**Comment by jackyoh** [comment]
retry

File: ohara-configurator/src/main/scala/com/island/ohara/configurator/route/StreamRoute.scala (hunk 1 above)
Comment (saivirtue) [reply ohara-configurator/src/main/scala/com/island/ohara/configurator/route/StreamRoute.scala]: yes. Will be implemented in OHARA-1363

File: ohara-client/src/main/scala/com/island/ohara/client/configurator/v0/StreamApi.scala (hunk 2 above)
Comment (saivirtue) [reply ohara-client/src/main/scala/com/island/ohara/client/configurator/v0/StreamApi.scala]: this class is useless and could be replace by `ContainerApi.ContainerState` .
Currently, I will use **RUNNING, EXITED, FAILED**

**Comment by saivirtue** [comment]
retry configurator

**Comment by saivirtue** [comment]
retry connector

**Pull request merged**